### ISSUE TITLE
PlatformTextButton is missing style

### ISSUE THREAD

**Issue by erperejildo** [stryder-dev/flutter_platform_widgets#336]
```
PlatformTextButton({
    this.widgetKey,
    this.onPressed,
    this.child,
    this.padding,
    this.alignment,
    this.material,
    this.cupertino,
  });
```

'PlatformButton' is deprecated and shouldn't be used. Use `PlatformTextButton`

How could I pass color?

<img width="305" alt="Screenshot 2021-12-10 at 12 16 01" src="https://user-images.githubusercontent.com/5629919/145565861-c9e08bea-eacc-4bb5-879c-b87415db4f62.png">

**Comment by aqwert** [comment]
The `PlatformTextButton` uses the not so new now material `TextButton` within Flutter. This is a replacement of the deprecated `FlatButton` and `RaisedButton` (now `ElevatedButton`) that the Flutter team has made. I added the deprecation annotation recently to reflect that these widgets by Flutter are also deprecated.

As for your particular situation if you want to change the color you can do the following.

```
PlatformTextButton(
    child: ...
    onPressed: ...
    material: (_, __) => MaterialTextButtonData(style:  TextButton.styleFrom(primary: Colors.red)),
    cupertino: (_, __) => CupertinoTextButtonData(color: Colors.red),
)
```

The thing is there are color properties on the `TextButton.styleFrom` static method which you may need to look at also.

Having said that, I can see the merit in offering a basic property to set the color that makes the above simpler. I will see if it can be done without too much compromise.

**Comment by erperejildo** [comment]
ok, let me know if that's not too complicated and if it is I can implement that solution

**Comment by aqwert** [comment]
Just published v1.12.0 that has a color property. Hope that helps

**Comment by erperejildo** [comment]
Thanks. I'll try it later
